apps/wallet/tests/staking.spec.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { expect, test } from './fixtures';
+import { createWallet } from './utils/auth';
+
+const TEST_TIMEOUT = 65 * 1000;
+const STAKE_AMOUNT = 100;
+
+test('staking', async ({ page, extensionUrl }) => {
+	test.setTimeout(TEST_TIMEOUT);
+
+	await createWallet(page, extensionUrl);
+
+	await page.getByTestId('faucet-request-button').click();
+	await expect(page.getByTestId('coin-balance')).not.toHaveText('0SUI');
+
+	await page.getByTestId('stake-and-earn-button').click();
```

Comment:
IS it possible to use role + text for some of these buttons instead of test IDs everywhere?